### PR TITLE
Update Auth samples to sync with internal changes

### PR DESCRIPTION
**Pull request by paulb777**

### CODE REVIEW

**Comment by renkelvin** [comment]
Could you give a brief description of this commit? I'm not familiar with this part of the code

**Comment by paulb777** [comment]
This is a mirror of cl/206829557

File: Example/Auth/ApiTests/FirebaseAuthApiTests.m [hunk 1]
```diff
@@ -23,8 +23,6 @@
 
 #ifdef NO_NETWORK
 #import "ITUIOSTestUtil.h"
-#import "ioReplayer/IORManager.h"
```
Comment (renkelvin) [inline Example/Auth/ApiTests/FirebaseAuthApiTests.m]: This line is not removed in the cl?

Comment (renkelvin) [reply Example/Auth/ApiTests/FirebaseAuthApiTests.m]: Yeah, I'll try to fix it.

**Pull request merged**